docker/newlib.cmake

```diff
@@ -0,0 +1,35 @@
+# toolchain file for newlib targets, see #1110
+
+# if we don't set generic, it tries to test the compiler which fails
```

Comment:
This comment needs to be changed: if we don't set it to generic, it will complain the system is unknown.